source/common/network/utility.cc

```diff
@@ -129,6 +129,71 @@ uint32_t Utility::portFromTcpUrl(const std::string& url) {
   }
 }
 
+Address::InstanceConstSharedPtr Utility::parseInternetAddress(const std::string& ip_address) {
+  sockaddr_in sa4;
+  if (inet_pton(AF_INET, ip_address.c_str(), &sa4.sin_addr) == 1) {
+    sa4.sin_family = AF_INET;
+    sa4.sin_port = 0;
+    return std::make_shared<Address::Ipv4Instance>(&sa4);
+  }
+  sockaddr_in6 sa6;
+  if (inet_pton(AF_INET6, ip_address.c_str(), &sa6.sin6_addr) == 1) {
+    sa6.sin6_family = AF_INET6;
+    sa6.sin6_port = 0;
+    return std::make_shared<Address::Ipv6Instance>(sa6);
+  }
+  throw EnvoyException(fmt::format("malformed IP address: {}", ip_address));
+}
+
+Address::InstanceConstSharedPtr
+Utility::parseInternetAddressAndPort(const std::string& ip_address) {
+  if (ip_address.empty()) {
+    Utility::throwWithMalformedIp(ip_address);
```

Comment:
nit: don't need to specify ```Utility::``` on these calls since already inside the class and static.